package.json

```
@@ -9,7 +9,15 @@
     "preview": "vite preview"
   },
   "devDependencies": {
+    "@babel/core": "^7.26.0",
+    "@babel/plugin-transform-react-jsx": "^7.25.9",
+    "@babel/plugin-transform-react-jsx-development": "^7.25.9",
+    "@babel/preset-react": "^7.26.3",
+    "@rollup/plugin-babel": "^6.0.4",
     "typescript": "~5.6.2",
     "vite": "^6.0.1"
+  },
+  "dependencies": {
+    "@vitejs/plugin-react": "^4.3.4"
```

```suggestion
    "@vitejs/plugin-react": "^4.3.4"
```
요 패키지는 `dependencies`에 설치하신 이유가 있으신가요??

---

요거는 실수인 것 같네요 ! :)